src/components/AutoComplete/AutoComplete.component.tsx

```diff
@@ -75,149 +75,156 @@ const getOptionsAsync = (
+        }
+    }, [open]);
 
-	/**
+    /**
 	 * A callback function called after searchQuery value is changed
 	 *
 	 * The function first deletes the previous option list and closes the popover list. Then,
 	 * calls the `getOptionAsync` function to get the new list according to the searchQuery
 	 * after the delay of 200ms.
 	 */
-	const getOptionsDelayed = useCallback(
-		debounce((query: string, callback: (options: ListItemType[]) => void) => {
-			setOptions([]);
-			setOpen(false);
-			getOptionsAsync(items, query).then(callback);
-		}, Constants.FETCH_FUNCTION_DELAY),
-		[]
-	);
+    const getOptionsDelayed = useCallback(
+        debounce((query: string, callback: (options: ListItemType[]) => void) => {
+            setOptions([]);
+            setOpen(false);
+            getOptionsAsync(items, query).then(callback);
+        }, Constants.FETCH_FUNCTION_DELAY),
+        [],
+    );
 
-	// On every change in searchQuery's value, it sets loading to true and
-	// fetch the new list according to the value of searchQuery
-	useEffect(() => {
-		setIsLoading(true);
+    // On every change in searchQuery's value, it sets loading to true and
+    // fetch the new list according to the value of searchQuery
+    useEffect(() => {
+        setIsLoading(true);
 
-		getOptionsDelayed(searchQuery, (options: ListItemType[]) => {
-			setOptions(options);
-			setOpen(Boolean(options.length && searchQuery));
-			setIsLoading(false);
-		});
-	}, [searchQuery, getOptionsDelayed]);
+        getOptionsDelayed(searchQuery, (options: ListItemType[]) => {
+            setOptions(options);
+            setOpen(Boolean(options.length && searchQuery));
+            setIsLoading(false);
+        });
+    }, [searchQuery, getOptionsDelayed]);
 
-	/**
+    /**
 	 * The function is called when an event is detected on the keyboard,
 	 * so you can browse through the list and select one.
 	 */
-	const onKeyDown: React.KeyboardEventHandler<HTMLInputElement> = (event) => {
-		if (event.key === 'Enter') {
-			const option = options[active];
-			setSearchValue(option.label ?? option.value);
-			setActive(0);
-			setOptions([]);
-			setOpen(false);
-			return;
-		}
-		if (event.key === 'ArrowUp') {
-			if (active === 0) return;
-			setActive((prev) => prev - 1);
-			listRef.current.scrollBy(0, -Constants.ITEM_HEIGHT);
-			return;
-		}
-		if (event.key === 'ArrowDown') {
-			if (active === options.length - 1) return;
-			setActive((prev) => prev + 1);
-			listRef.current.scrollBy(0, Constants.ITEM_HEIGHT);
-			return;
-		}
-	};
+    const onKeyDown: React.KeyboardEventHandler<HTMLInputElement> = (event) => {
+        if (event.key === 'Enter') {
+            const option = options[active];
+            setSearchValue(option.label ?? option.value);
+            setActive(0);
+            setOptions([]);
+            setOpen(false);
+            return;
+        }
+        if (event.key === 'ArrowUp') {
+            if (active === 0) {
```

Comment:
```suggestion
            if (!active) {
```

Reply:
This is required it needs to be equal to 0 only